tests/remote_config/rc_expected_requests_asm_features.json

```diff
@@ -438,14 +529,25 @@
 						"hash": "9221dfd9f6084151313e3e4920121ae843614c328e4630ea371ba66e2f15a0a6"
 					}
 				]
+			},
+			{
+				"path": "datadog/2/ASM_FEATURES/ASM_FEATURES-third/testname",
+				"length": 41,
+				"hashes": [
+					{
+						"algorithm": "sha256",
+						"hash": "1e4a75edfd5f65e0adec704cc7c32f79987117d84755544ae4905045cdb0a443"
+					}
+				]
 			}
-		]
+		],
+		"test_description": "ERROR - File missing in TUF targets - Error mode of RFC section 3d. RFC about integrating with remote-config: https://docs.google.com/document/d/1u_G7TOr8wJX0dOM_zUDKuRJgxoJU_hVTd5SeaMucQUs"
 	},
 	{
 		"client": {
 			"state": {
 				"root_version": 1,
-				"targets_version": 11,
+				"targets_version": 13,
```

Comment:
if the previous test reports an error, the target version should be "targets_version": 12, right?